ledger/lrukv.go

```diff
@@ -79,7 +79,7 @@ func (m *lruKV) read(key string) (data trackerdb.PersistedKVData, has bool) {
 func (m *lruKV) flushPendingWrites() {
 	pendingEntriesCount := len(m.pendingKVs)
 	if pendingEntriesCount >= m.pendingWritesWarnThreshold {
-		m.log.Warnf("lruKV: number of entries in pendingKVs(%d) exceed the warning threshold of %d", pendingEntriesCount, m.pendingWritesWarnThreshold)
+		m.log.Infof("lruKV: number of entries in pendingKVs(%d) exceed the warning threshold of %d", pendingEntriesCount, m.pendingWritesWarnThreshold)
```

Comment:
After a brief investigation into the usage of `Warnf` vs. `Infof` in this repo, I support this change. `Warnf` seems to be used in cases when an error condition actually results in a different codepath. On the other hand, in this case no action is taken other than printing a log because of the problematic state.

Reply:
Thanks! I also checked with Carbon-11 regarding these specific warnings.